internal/proto/local/controller/api/services/v1/account_service.proto

```diff
@@ -124,6 +124,7 @@ message GetAccountResponse {
 
 message ListAccountsRequest {
   string auth_method_id = 1 [json_name="auth_method_id"];
+  string filter = 2;
```

Comment:
Since filter is 30 in a lot of the files below, could we just standardize on it as being 30 everywhere? It's 1 here and 20 in a couple of places. Just seems nicer, although I know it doesn't matter in a technical sense.

Reply:
I've updated the protos to try to be consistent with making filter be 10s based field with a json_name option specified.  Like you said, it doesn't matter technically but consistency is important.